lib/contentful/delivery.ex

```diff
@@ -21,14 +27,19 @@ defmodule Contentful.Delivery do
       entries_url,
       access_token,
       params
-    ) |> resolve_includes
+    )
 
     response["items"]
   end
 
   def entry(space_id, access_token, entry_id, params \\ %{}) do
-    entries(space_id, access_token, Dict.merge(params, %{'sys.id' => entry_id}))
-      |> Enum.fetch(0)
+    entry_url = "/spaces/#{space_id}/entries/#{entry_id}"
```

Comment:
This was explicitly done this way, because of include resolution, which does not happen in single entry queries.